docs/user_guide/wrappers/Wrapper.rst

```diff
@@ -116,6 +116,56 @@ to select only a subset of the variables.
     X_train_t = selector.transform(X_train.fillna(0))
     X_test_t = selector.transform(X_test.fillna(0))
 
+Even though Feature Engine has its own implementation of OneHotEncoder, you may want to use Scikit-Learn Implementation in order to access different options, such as drop first Category.
+
+.. code:: python
+
+    import pandas as pd
+    import numpy as np
+    from sklearn.model_selection import train_test_split
+    from sklearn.preprocessing import OneHotEncoder
+
+    df = pd.read_csv('https://www.openml.org/data/get_csv/16826755/phpMYEkMl')
+    X = df[['pclass']]
+    y = df.survived
+    X_train, X_test, y_train, y_test= train_test_split(X, y, test_size=0.2, random_state=42)
+
+    ohe = SklearnTransformerWrapper(OneHotEncoder(sparse=False, drop='first'))
+
+    ohe.fit(X_train)
+
+    X_train_transformed = ohe.transform(X_train)
+    X_test_transformed = ohe.transform(X_test)
+
+
+Let's say you want to use :class:`SklearnTransformerWrapper()` in a more complex context. As you may note there are `?` signs to denote unknown values. Due to the complexity of the transformations needed we'll use a Pipeline to impute missing values, encode categorical features and create interactions for specific variables using Scikit-Learn PolynomialFeatures.
```

Comment:
It would be great if you could shorten the length of the sentence to 88 char (that is, break the sentence over 2 lines). 

It reads well, the last 2 words should be: Scikit-Learn's PolynomialFeatures().